### PR TITLE
#133 unit tests

### PR DESCRIPTION
**Pull request by ordd**
# Description

Some tests use edge case numbers like "1" GD, which can not be divided.

About # (link your issue here)
#133 

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:
- [ ] PR title matches follow: (Feature|Bug|Chore) Task Name
- [ ] My code follows the style guidelines of this project
- [ ] I have followed all the instructions described in the initial task (check Definitions of Done)
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added reference to a related issue in the repository
- [ ] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: stakingModel/test/GoodReserveCDai.test.js [hunk 1]
```diff
@@ -332,33 +363,29 @@ contract("GoodReserve - staking with cDAI mocks", ([founder, staker]) => {
...
     let transaction = await goodReserve.sell(cDAI.address, amount, 0);
     reserveToken = await marketMaker.reserveTokens(cDAI.address);
     let reserveBalanceAfter = reserveToken.reserveSupply;
     let supplyAfter = reserveToken.gdSupply;
-    let rrAfter = reserveToken.reserveRatio;
     const gdBalanceAfter = await goodDollar.balanceOf(founder);
     const cDAIBalanceAfter = await cDAI.balanceOf(founder);
     const cDAIBalanceReserveAfter = await cDAI.balanceOf(goodReserve.address);
-    const priceAfter = await goodReserve.currentPrice(cDAI.address);
+    // according to the initialization settings we should expect 1000000
```
Comment (sirpy) [inline stakingModel/test/GoodReserveCDai.test.js]: this says nothing. it is exactly what the test itself says "we are expecting 100000" but you are not explaining why 100000
how did you reach this number?

Comment (ordd) [reply stakingModel/test/GoodReserveCDai.test.js]: d94eca8064aa0ed7e41c368365fad416f731edc5
